projects/lib/src/board/board.cpp

```diff
@@ -319,14 +319,15 @@ QString Board::moveString(const Move& move, MoveNotation notation)
 	return lanMoveString(move);
 }
 
-Move Board::moveFromLanString(const QString& str)
+Move Board::moveFromLanString(const QString& istr)
 {
+	QString str(istr);
+	str.remove(QRegExp("[x+#!?]"));
```

Comment:
Of course you reacted to this. This touches a topic that needs some clarification for various reasons.

A more lenient reader is somewhat slower but may have advantages. Actually this is a *work-around* as long as the Berolina SAN notation is not agreed upon. The cutechess SAN reader does not like the longer pawn notation e2c4 and c5xc4  (which is Xboard-like) instead of ec4 and cxc4. So in the Board::moveFromString it will not get a valid moveFromSanString result and thus fall back and try moveFromLanString where a capture notation c5xc4 will spoil the result. To be able to read Xboard-scored Berolina games I made this change for now.

To resolve this I suggest to clarify two topics:

1. Berolina notation standards. I would prefer to have a common basis with authors of relevant software how to handle this.
2. Notation reader standards of cutechess. On the one hand the written and exported notations have to be strict and correct, but what is the goal of the reading methods? Should they be lenient and able to read from various sources or are they designed to be very strict and cough on small deviations?

Regarding the work-around, a correct SAN parser would find out about ambiguous source fields that result in a pawn move to the parsed target field. I have left that work out because I did not want to put effort into this before getting a  response from you.